ai/README.md

```diff
@@ -0,0 +1,853 @@
+# Behavioral Question
+- Standard QB Behaviroal question
+- What is the bias variance tradeoff? If an engineer said that they have a high bias and low variance in their classification model, what would you advice the junior to do.
+
+# Portfolio Presentation
+Please present 1 Machine Learning preferably one that involes LLM you have done in the past. Explain what the problem was and how you accomplished the solution.
+
+
+# Coding Exercise
+The purpose of this exercise is not to give a "gotcha" question or puzzle but a straightforward (albeit contrived) example of the requirement that might arise in an actual project so that we have shared context for a technical conversation during the interview. We are interested in how you approach a project. How you demonstrate the correctness of your implementation is up to you.
+
+
+## Requirements
+
+### Goal
+Create a command-line Python program that compares semantic searches based on embeddings. 
+
+- Below is the  “data set” of 2 applications. “: CRM App and Inventory App
+- Generate embeddings for both datasets and save them. Given a new JSON, compare the embeddings and return the best match.
+- Please use Python 3. x
+- You can use whatever library you prefer to generate embedding. But as a stretch, you can use AWS Bedrock to demonstrate your cloud knowledge.
+
+
+
+### Need
+- Please use best practices from Python.
+- Please provide unit tests for the main program functionality. Create a separate module for the unit tests.
```

Comment:
For the other coding exercises we don't explicitly ask for unit tests, we let the candidate decide on how / if to test. Is this here because we are looking for a senior person?